pkgs/development/node-packages/node-packages-v8.json

```diff
@@ -14,6 +14,7 @@
 , "sloc"
 , "vue-cli"
```

Comment:
So `vue-cli` is the old `@vue/cli`?

Reply:
@vue/cli is Vue CLI v3
vue-cli is Vue CLI v2

Even though v2 is deprecated, most users still use it, we must give them enough time to switch...

Reply:
ok. is it possible to add it to `node-packages-v6`?

Reply:
From the vue-cli v3 documentation: "Vue CLI requires Node.js version 8.9"